README.md

```diff
@@ -1,45 +1,191 @@
 # module-manager
 
-This repository offers an operator and relevant packages to perform installation, update and uninstallation of helm charts.
-The packages could be used independent of the operator.
+The [module-manager](#module-manager) plays an important part in the modularization ecosystem to handle installation of resources from control-plane to runtime clusters. 
+For more information, refer to the [architecture summary](https://github.com/kyma-project/lifecycle-manager#architecture).
+This repository offers an **_operator_** and relevant **_library packages_** to perform reconciliation of helm chart resources, configurations and custom resource state handling.
+The **_library package_** could be used independently of the operator.
 
-**Disclaimer**: This repository is still in development and implementation details can change quickly.
+### Content
```

Comment:
If we have a content lineout, we should also do the same on Lifecycle Manager

Reply:
https://github.com/kyma-project/lifecycle-manager/pull/250